### PR TITLE
WIP: #12568. Delay syncdown for syncs when uploading folder detects a recursivity

### PR DESCRIPTION
**Pull request by polmr**

### CODE REVIEW

**Comment by polmr** [comment]
trigger compilation

File: src/megaapi_impl.cpp [hunk 1]
```diff
@@ -1147,6 +1147,46 @@ MegaSync *MegaApiImpl::getSyncByPath(const char *localPath)
...
+{
+    int toret = -1;
+    if (!node)
+    {
+        return toret;
+    }
+
+    sdkMutex.lock();
+    Node *n = client->nodebyhandle(node->getHandle());
+    if (n->localnode && n->localnode->sync)
```
Comment (polmr) [inline src/megaapi_impl.cpp]: should n!=NULL be controlled here?

**Comment by polmr** [comment]
trigger compilation

**Comment by sergiohs84** [comment]
Better to wait for the completion of https://github.com/meganz/sdk/pull/1510 first. Afterwards, this branch can be updated and it will be ready for review+merge.

**Pull request closed without merging**